Shared/R4/resources/Library-R4-StressEchoCommon-prepopulation.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Should these shred cql and json files go into the folder for the rule if they are only used by the one rule and will unlikely be used by any others that we write?